models/evc.py

```diff
@@ -1410,6 +1410,45 @@ def handle_link_down(self):
 
         return success
 
+    @staticmethod
+    def get_interface_from_switch(uni: UNI, switches: dict) -> Interface:
+        """Get interface from switch by uni"""
+        switch = switches[uni.interface.switch.dpid]
+        interface = switch.interfaces[uni.interface.port_number]
+        return interface
+
+    def handle_topology_update(self, switches: dict):
+        """Handle changes in the topology"""
+        interface_a = self.get_interface_from_switch(self.uni_a, switches)
+        interface_z = self.get_interface_from_switch(self.uni_z, switches)
+        active = self.is_uni_interface_active(interface_a, interface_z)
+
+        if self.is_active() != active:
```

Comment:
It'd be great to have `log.info` messages informing that a given EVC isn't active due to UNI "A" or "Z being down (and specifically which one), and similarly, when it goes up due to an UNI to also `log.info` message.

Reply:
Added logs